tools/java/org/unicode/cldr/tool/GeneratedPluralSamples.java

```diff
@@ -28,7 +28,6 @@
 import com.google.common.base.Joiner;
```

Comment:
Here are the main changes, moving to EFixedPoint. Had to break out the samples into different groups. I first tried to merge with the range notation, but that got to be really ugly.